package.json

```diff
@@ -27,6 +27,9 @@
     "falcor-express": "^0.1.2"
   },
   "devDependencies": {
-    "eslint": "^1.7.1"
+    "eslint": "^1.7.1",
+    "browser-sync": "^2.8.2",
```

Comment:
We are not using `browser-sync` anymore. Better to remove this from here. I think i removed this in last commit.